grammars/hrm.cson

```diff
@@ -32,29 +32,25 @@
         'name': 'keyword.control.c'
   }
   {
-    'match': '^    (COMMENT) +([0-9]+)'
+    'match': '^ *(COMMENT) +([0-9]+)'
     'captures':
       '1':
         'name': 'entity.name.function.preprocessor.c'
       '2':
-        'name': 'string.quoted.double.include.c'
+        'name': 'constant.numeric.c'
```

Comment:
I'd probably simply use type 'variable' or 'string'. If this weren't assembly, 'string' would make the most sense. Using default atom colors, string is green while variable is a darker red/orange. String looks better imo.

http://imgur.com/6ihPM9p